### ISSUE TITLE
Limit pitchfork command to developers

### ISSUE THREAD

**Issue by jayvdb** [coala/corobo#358]
https://github.com/coala/corobo/issues/36 requested "When issued by an authorized person to cobot,.." , however that bit wasnt done.
This needs to be fixed before GCI or there will be mayhem when this easter egg is found.

**Comment by aashraybhandar1** [comment]
@jayvdb  How do I check if the person issuing the command is a developer? Do I have to use labhub to do the same?

**Comment by aashraybhandar1** [comment]
Also should pitchfork be part of labhub ?? @jayvdb

**Comment by jayvdb** [comment]
Yes, the data you need is in labhub, so the pitchfork plugin needs to *depend on* labhub, and get the data from there.  pitchfork should not be part of labhub

**Comment by Vamshi99** [comment]
ping @aashraybhandar1 Are you working on this? If not, let us know.

**Comment by aashraybhandar1** [comment]
Hey I'm really sorry.I had my end semester exams so wasn't able to continue working on it. I do want to work on this issue and I will try to wrap it up in the and atleast send the first pull request in the next couple of days.

**Comment by jayvdb** [comment]
Turns out this is hard due to the problem of testing dependent plugins

**Comment by jayvdb** [comment]
This needs to be implemented as a filter to prevent access to any command based on configuration.
